A32NX/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -401,4 +416,78 @@ class A32NX_FWC {
         }
         return true;
     }
+
+    _autopilotDisconnect(_deltaTime) {
+        const apStatus = SimVar.GetSimVarValue("AUTOPILOT MASTER", "Bool");
+        const atherStatus = SimVar.GetSimVarValue("AUTOTHROTTLE ACTIVE", "Bool");
+
+        if (atherStatus === 1) {
+            SimVar.SetSimVarValue("L:A32NX_ATHR_DISC", "Bool", false);
+            SimVar.SetSimVarValue("L:Generic_Master_Caution_Active", "Bool", false);
+            this.AutothrottleWarningCanceled = false;
+            this.athrdeltaTime = 0;
+        }
+
+        if (atherStatus === 0 && this.AutothrottleWarningCanceled === false) {
+            SimVar.SetSimVarValue("L:A32NX_ATHR_DISC", "Bool", true);
+            SimVar.SetSimVarValue("L:Generic_Master_Caution_Active", "Bool", true);
+            this.athrdeltaTime += _deltaTime;
+            if (this.cautionPressed === true || (this.athrdeltaTime / 1000) >= 3) {
+                this.AutothrottleWarningCanceled = true;
+                SimVar.SetSimVarValue("L:A32NX_ATHR_DISC", "Bool", false);
+                SimVar.SetSimVarValue("L:Generic_Master_Caution_Active", "Bool", false);
+                this.athrdeltaTime = 0;
+            }
+        }
+
+        if (apStatus === 1) {
+            SimVar.SetSimVarValue("L:A32NX_AP_DISC", "Bool", false);
+            SimVar.SetSimVarValue("L:Generic_Master_Warning_Active", "Bool", false);
+            this.AutopilotWarningCanceled = false;
+            this.apdeltaTime = 0;
+        }
+
+        if (apStatus === 0 && this.AutopilotWarningCanceled === false) {
+            SimVar.SetSimVarValue("L:A32NX_AP_DISC", "Bool", true);
+            SimVar.SetSimVarValue("L:Generic_Master_Warning_Active", "Bool", true);
+            this.apdeltaTime += _deltaTime;
+            if (this.warningPressed === true || (this.apdeltaTime / 1000) >= 3) {
+                this.AutopilotWarningCanceled = true;
+                SimVar.SetSimVarValue("L:A32NX_AP_DISC", "Bool", false);
+                SimVar.SetSimVarValue("L:Generic_Master_Warning_Active", "Bool", false);
+                this.apdeltaTime = 0;
+            }
+        }
+    }
+
+    _checkLandingGear(_deltaTime) {
```

Comment:
This does not need to be run every single tick. especially as it always access all these SimVars. How about we use the update throttler class to only run these specific checks once or twice per second?

Reply:
Ok!

Reply:
I added it but idk that is correct. can you check it?

Reply:
That looks correct! Does it still work?